content/en/observability_pipelines/sources/opentelemetry.md

```diff
@@ -73,6 +77,44 @@ Based on these example configurations, these are values you enter for the follow
 - HTTP listener address: `worker:4317`
 - gRPC listener address: `worker:4318`
 
+## Send logs from the Datadog Distribution of OpenTelemetry Collector to Observability Pipelines
+
+To send logs from the Datadog Distribution of the OpenTelemetry Collector (DDOT):
+1. Deploy the DDOT Collector using helm. See [Install the DDOT Collector as a Kubernetes DaemonSet][5] for instructions.
+1. [Set up a pipeline][6] on Observability Pipelines using the [OpenTelemetry source](#set-up-the-source-in-the-pipeline-ui).
+    1. (Optional) Datadog recommends adding an [Edit Fields processor][7] to the pipeline that appends the field `op_otel_ddot:true`.
+    1. When you install the Worker, for the OpenTelemetry source environment variables:
+        1. Set your HTTP listener to `0.0.0.0:4318`.
+        1. Set your gRPC listener to `0.0.0.0:4317`.
+    1. After you installed the Worker and deployed the pipeline, update the OpenTelemetry Collector's `collector-config.yaml` to include an exporter that sends logs to Observability Pipelines. For example:
+        ```
+        exporters:
+            datadog:
+                api:
+                site: ${env:<DD_SITE>}
+                key: ${env:<DD_API_KEY>}
+            otlphttp:
+                endpoint: http://opw-observability-pipelines-worker.default.svc.cluster.local:4318
+        ```
+    1. Redeploy the Datadog Agent with the updated `collector-config.yaml`.For example, if the Agent is installed in Kubernetes:
+        ```
+        helm upgrade --install datadog-agent datadog/datadog \
+        --values ./agent.yaml \
+        --set-file datadog.otelCollector.config=./collector-config.yaml
+        ```
+
+**Notes**:
+- These settings are used when setting up the Datadog Agent source in Observability Pipelines:
+    - `DD_OBSERVABILITY_PIPELINES_WORKER_LOGS_ENABLED:true`
```

Comment:
We can probably just leave it `DD_OBSERVABILITY_PIPELINES_WORKER_LOGS_ENABLED` and `DD_OBSERVABILITY_PIPELINES_WORKER_LOGS_URL` without the values I think, WDYT?